day1/ques4.rb

```diff
@@ -0,0 +1,81 @@
+# 4. Regex for mobile number, email address, name, gender(M / F) and  amount
+
+def validNumber(number)
```

Comment:
spacing and indentations no proper add some extension which can help formating the code